subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn.c

```diff
@@ -310,11 +340,11 @@ void lll_conn_isr_rx(void *param)
 	radio_gpio_pa_setup();
 
 #if defined(CONFIG_BT_CTLR_PHY)
-	radio_gpio_pa_lna_enable(radio_tmr_tifs_base_get() + EVENT_IFS_US -
+	radio_gpio_pa_lna_enable(radio_tmr_tifs_base_get() + EVENT_IFS_US + cte_len -
 				 radio_rx_chain_delay_get(lll->phy_rx, 1) -
 				 HAL_RADIO_GPIO_PA_OFFSET);
 #else /* !CONFIG_BT_CTLR_PHY */
-	radio_gpio_pa_lna_enable(radio_tmr_tifs_base_get() + EVENT_IFS_US -
+	radio_gpio_pa_lna_enable(radio_tmr_tifs_base_get() + EVENT_IFS_US + cte_len -
```

Comment:
`radio_tmr_tifs_base_get() + EVENT_IFS_US + cte_len` and `- HAL_RADIO_GPIO_PA_OFFSET` is common between the two calls to `radio_gpio_pa_lna_enable`. We could consider moving the common calculation to a common variable, and only change the input of `radio_rx_chain_delay_get` for these calls.

Reply:
Changes added